pages/linux/setquota.md

```diff
@@ -0,0 +1,24 @@
+# setquota
+
+> Set disk quotas for a user or group.
+> More information: <https://manned.org/setquota.8>.
+
+- Edit remote quota over RPC, uses rpc.quotad on the remote host to set quota:
+
+`setquota -r, --remote`
```

Comment:
```suggestion
`setquota --remote`
```

- Mandatory fix: [invalid](https://manned.org/setquota.8) command syntax: there is no `-r,` option: man pages can't be often copied without any modifications in TlDr pages.
- Note: long and short options can't be presented now without using placeholder syntax.